Makefile

```diff
@@ -106,12 +106,14 @@ test/test: ${OBJ} ${TEST_OBJ}
 	${CC} -o ${@} ${TEST_OBJ} $(filter-out ${TEST_OBJ:test/%=src/%},${OBJ}) ${CFLAGS} ${LDFLAGS}
 
 .PHONY: doc doc-doxygen
-doc: docs/dunst.1 docs/dunstctl.1
```

Comment:
Since, you're adding another manpage, you have to add it to the install targets. It's currently missing in the target `install-dunst`.

Reply:
Added it. While I was at it, I added it to the uninstall and test-install.sh as well.